Few minor changes (and Wiki-Links) (#1071)
Heegu-sama/Homm3BG: sections/units.tex

\textbf{Reinforcing} requires that your Town has a Citadel\index{Citadel} in addition to a Dwelling of that unit's Level.\par

\note{13}{

>   Any effects which instruct you to reinforce (e.g. \wikilink{abilities/necromancy}{Necromancy}) \textbf{do not} require spending your Population Token nor owning a Citadel or any Dwellings.\par

Reviewer: ```suggestion
  Any effects which instruct you to reinforce (e.g., \wikilink{abilities/necromancy}{Necromancy}) \textbf{do not} require spending your Population Token nor owning a Citadel or any Dwellings.\par
```

    It replaces all movement and/or attacking.
  \item\textbf{Passive} \svg{unit_passive} resolves whenever its condition is met.
  \item\textbf{Retaliate} \svg{unit_retaliation} resolves when the unit retaliates.

>   \item\textbf{Map} \svg{map} effects can be activated once per turn outside of combat as long as you have this Unit in your Unit Deck.

Reviewer: ```suggestion
  \item\textbf{Map} \svg{map} effects may be resolved once per Turn outside of Combat as long as a unit with this effect is in your Unit Deck.
```
